internal/drand-cli/dkg_cli.go

```diff
@@ -552,21 +516,20 @@ func csvPrint(c *cli.Context, tag string, entry *drand.DKGEntry) {
 }
 
 type printModel struct {
```

Comment:
I'm not convinced this is really required to get a pretty printer xD But sure, why not

Reply:
it's not, but the alternative of procedurally going through the object and handling all the fields for printing was laborious